CONTRIBUTING

```diff
@@ -0,0 +1,102 @@
+# How to contribute
+
+Welcome to `digits` library for GoLang.
+
+###### TODO <@joshuanario, insert blurb here about ECKSBEE and the library.>
+###### TODO <Contact Information / Slack Invite>
+
+### Resources
+
+| Resource | Description |
+| --- | --- |
+| [XBRL Standards](https://www.xbrl.org/) | Business Reporting Standard Community. Details XBRL and how it is used, and what to expect with this library.       |
+| [EDGAR Browser](https://edgarbrowser.com) | Open Source EDGAR Browser, leveraging Telefacts. |
+| [Telefacts](https://github.com/ecksbee/telefacts) | XBRL file storage, data types, and renderer. |
+| [ECKSBEE](https://ecksbee.com/) | Our |
+
+###### TODO <consider adding other resources, like were bugs should be reported to, etc.>
+
+## Coding conventions
+
+Start reading our code and you'll get the hang of it. This library is optimized for size and efficiency:
+
+1. `digits.go` contains the only public facing functionality.
+2. Use tabs (one character vs multiple characters for spaces)
+3. Put spaces after list items and method parameters (`[1, 2, 3]`, not `[1,2,3]`), around operators (`x += 1`, not `x+=1`), and around hash arrows.
+4. Use descriptive variable names. Make the variables intent/use clear.
+    - Indexing variables from loops are fine as single characters (`i`, `j`, `k`, etc)
+5. All commits must compile. If you have Work-In-Progress(WIP) code, then ensure it is in a state that can compile before committing.
+6. No inline comments in release branch code, except for `TODO` statements marking bugs or next steps.
+    - Function documentation is acceptable.
+7. This is open source software. Consider the people who will read your code, and make sure intent is clear.
+    - If something seems convoluted enough to warrant comments, use function documentation.

```

Comment:
This is open source software. Consider the people who will read your code, and make sure intent is clear.
    - If something seems convoluted enough to warrant comments, use function documentation.

I don't like this statement.  Personally, I write code knowing that the whole world can read it (in reality only my temmates will/can read it).  The point I'm saying is not everybody writes code for readability.  Functionality is definitely a #1 priority for every computer programmer.  Reliability should be a strong #2.  Something like readability is not a priority...and that's why we do code reviews to ensure that the code can be maintained by other devs and perhaps gain insight what others view as readable vs unreadable.

Bottomline, please remove this, and let's make sure we have a good code review process.